src/protocol/requests/listGroups/v2/request.js

```diff
@@ -0,0 +1,7 @@
+const requestV0 = require('../v0/request')
+
+/**
+ * ListGroups Request (Version: 1)
+ */
+
+module.exports = () => Object.assign(requestV0(), { apiVersion: 1 })
```

Comment:
This should be `apiVersion: 2`. Probably more copy-paste mistake. Same with the comment above.

Although it shouldn't really matter in this case, since the request looks the same, I think it would make sense for this to import `v1`, even if v1 just delegates to v0.